app/graphql/mutations/profile/create.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module Mutations
+  module Profile
+    # Mutation to create a Profile
+    class Create < BaseMutation
+      graphql_name 'createProfile'
+
+      argument :clone_from_profile_id, ID, required: true
+      argument :business_objective_id, ID, required: false
+      argument :benchmark_id, ID, required: true
+      argument :name, String, required: true
+      argument :ref_id, ID, required: true
+      argument :description, String, required: false
+      argument :business_objective_id, String, required: false
+      argument :compliance_threshold, Float, required: false
+      field :profile, Types::Profile, null: false
+
+      def resolve(args = {})
+        original_profile = find_original_profile(args[:clone_from_profile_id])
+        profile = ::Profile.new(new_profile_options(args))
+        profile.save
+        profile.add_rules_from(profile: original_profile)
```

Comment:
I tried for a bit reusing `clone_to` here, but it was more complicated than I thought. 
What I did was changing `clone_to` to take in a hash of all possible arguments to Profile (in this case `new_profile_options`). This worked for the regular use case of parsing XCCDF reports. However `clone_to` expects that either the cloned profile or the clonee is `nil` to update the attributes (https://github.com/RedHatInsights/compliance-backend/blob/master/app/models/profile.rb#L92 , I changed it to use all attributes).    

In any case, the suggestion was good, it allowed me to remove the method I had in the original commit which was almost a copy of `Profile#add_rules_from` :sweat:  thanks for noticing that

Reply:
So, correct me if I'm wrong, but this mutation will raise a 500 error if the user tries to create the same profile twice, right? I guess `clone_to` is meant to try and find an existing profile first before creating a new one, and that's not functionality we want in this mutation.

Reply:
It raised a 500, correct - I have changed `save` to `save!`- this returns a 422 if you try to create the same profile twice. I have also seen we can catch these with:
```
.catch(
            (err) => {
                <ErrorPage error={err}/>;
            }
        );
```
on the mutation on the wizard, which I'm working on